tests/syntax_tests/data/printer/expr/expected/braced.res.txt

```diff
@@ -318,5 +318,5 @@ let x = {
 }
 
 // string constant should be printed correct
-(<> {"\n"->React.string} </>)
-(<> {"\""->React.string} </>)
+<> {"\n"->React.string} </>
```

Comment:
@shulhi is this change a good one? I don't think there is harm that the `(` `)` are gone here? Let me know if this needs fixing.

Reply:
I'm okay with this.